main.js

```diff
@@ -30,6 +31,19 @@
     clearButton.setAttribute('disabled', true);
   }
 
+  function resetGame() {
```

Comment:
this is iteration 1.6 that resets the guess form when the winner is declared and resetting the min range to 0 and max range to 100 and then clears the fields for the next game